chapters/development_tools.md

```diff
@@ -16,6 +16,8 @@ Besides the Validation Layers, there are also other publicly available layers th
 
 - [API Logging](https://vulkan.lunarg.com/doc/sdk/latest/windows/api_dump_layer.html) - Vulkan SDK layer for logging API calls.
 - [Arm PerfDoc layer](https://github.com/ARM-software/perfdoc) - Checks Vulkan applications for best practices on Arm Mali devices.
+- [PowerVR PerfDoc layer](https://github.com/powervr-graphics/perfdoc) - Checks Vulkan applications for best practices on Imagination Technologies PowerVR devices.
+- [Vulkan Adreno Layer](https://developer.qualcomm.com/software/adreno-gpu-sdk/tools) - Checks Vulkan applications for best practices on Qualcomm Adreno devices.
```

Comment:
Was just about to say this wasn't an open public link in previous MR

The only nit is I tried to just make this list alphabetic, so if you could quick reorder that would be awesome

Reply:
Done.